provider/coredns.go

```diff
@@ -258,6 +258,28 @@ func NewCoreDNSProvider(domainFilter endpoint.DomainFilter, prefix string, dryRu
 	}, nil
 }
 
+// Find takes a Endpoint slice and looks for an element in it. If found it will
+// return it's key, otherwise it will return -1 and a bool of false.
+func findEp(slice []*endpoint.Endpoint, dnsName string) (int, bool) {
+	for i, item := range slice {
+		if item.DNSName == dnsName {
+			return i, true
+		}
+	}
+	return -1, false
+}
+
+// Find takes a ep.Targets string slice and looks for an element in it. If found it will
+// return it's key, otherwise it will return -1 and a bool of false.
```

Comment:
The code comment is wrong, this was likely a copy-paste, the function returns a string.

Reply:
@Raffo  thanks for the catch, comments are fixed